packages/amplify-category-analytics/src/provider-utils/awscloudformation/service-walkthroughs/pinpoint-walkthrough.ts

```diff
@@ -38,6 +38,7 @@ export const addWalkthrough = async (context : $TSContext, defaultValuesFilename
   } else {
     return configure(context, defaultValuesFilename, serviceMetadata, undefined);
   }
+  return undefined
```

Comment:
I would expect the lint rules to add a missing semi-colon automatically, are we missing a rule?

Reply:
It will. This was my own omission. I was trying to avoid running the formatter to minimize the commit noise. I've formatted this file in the latest commit.